.github/stale.yml

```diff
@@ -0,0 +1,62 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - "contrib: maybe good first bug"
+  - "contrib: good first bug"
+  - "contrib: welcome"
+  - "component: security"
```

Comment:
my preference would be against a straight "neverstale" label because it doesn't explain _why_ we don't want it never to be stale, but perhaps there are other labels we could add to the list that would be used for issues we really want to fix... eventually.  I thought about 'priority: p2` but those should never be left alone for 60 days.  

Finally a reason for the `priority: p2.5` label 🤣

Reply:
Or, we just have some ongoing projects papercuts and other things we're intending to fix that we can add the issues to to keep them non-stale.  Mmm, yes, I think I'd prefer that.

Reply:
> Or, we just have some ongoing projects papercuts and other things we're intending to fix that we can add the issues to to keep them non-stale. Mmm, yes, I think I'd prefer that.

So you want "type: papercut" added here, any others?

Reply:
> my preference would be against a straight "neverstale" label because it doesn't explain _why_ we don't want it never to be stale, but perhaps there are other labels we could add to the list that would be used for issues we really want to fix... eventually. I thought about 'priority: p2` but those should never be left alone for 60 days.
> 
> Finally a reason for the `priority: p2.5` label 🤣

The use-case for an explicit `neverstale` label of some kind is having a direct way to tell stalebot to stay out of the way. It wouldn't always be the case that a label in `exemptLabels` would be a good fit. If that label is used then I'd expect the comments to provide a rationale.

Reply:
> So you want "type: papercut" added here, any others?

no, I mean we'd have actual project to collect issues we specifically care about and are intending to fix.  Issues in projects are already excluded according the config.

Reply:
> > So you want "type: papercut" added here, any others?
> 
> no, I mean we'd have actual project to collect issues we specifically care about and are intending to fix. Issues in projects are already excluded according the config.

Ah ok, I'll pull that.